frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.tsx

```diff
@@ -107,77 +94,167 @@ const DashboardInfoSidebarBody = ({
+  setPage: (
+    page: "default" | "caching" | SetStateAction<"default" | "caching">,
+  ) => void;
+  showCaching: boolean;
+}) => {
+  const isCacheable = isDashboardCacheable(dashboard);
+  const autoApplyFilterToggleId = useUniqueId();
+  const dispatch = useDispatch();
   const handleToggleAutoApplyFilters = useCallback(
     (isAutoApplyingFilters: boolean) => {
       dispatch(toggleAutoApplyFilters(isAutoApplyingFilters));
     },
     [dispatch],
   );
 
-  const autoApplyFilterToggleId = useUniqueId();
-  const canWrite = dashboard.can_write && !dashboard.archived;
-  const isCacheable = isDashboardCacheable(dashboard);
-
-  const showCaching = canWrite && PLUGIN_CACHING.isGranularCachingEnabled();
-
   return (
-    <>
-      <ContentSection>
-        <DescriptionHeader>{t`About`}</DescriptionHeader>
-        <EditableDescription
-          initialValue={dashboard.description}
-          isDisabled={!canWrite}
-          onChange={handleDescriptionChange}
-          onFocus={() => setDescriptionError("")}
-          onBlur={handleDescriptionBlur}
-          isOptional
-          isMultiline
-          isMarkdown
-          hasError={!!descriptionError}
-          placeholder={t`Add description`}
-          key={`dashboard-description-${dashboard.description}`}
-          style={{ fontSize: ".875rem" }}
-        />
+    <Stack spacing="lg">
+      <SidesheetCard title={t`Description`} pb="md">
+        <div className={DashboardInfoSidebarS.EditableTextContainer}>
+          <EditableText
+            initialValue={dashboard.description}
+            isDisabled={!canWrite}
+            onChange={handleDescriptionChange}
+            onFocus={() => setDescriptionError("")}
+            onBlur={handleDescriptionBlur}
+            isOptional
+            isMultiline
+            isMarkdown
+            placeholder={t`Add description`}
+          />
+        </div>
         {!!descriptionError && (
           <Text color="error" size="xs" mt="xs">
             {descriptionError}
           </Text>
         )}
-      </ContentSection>
+      </SidesheetCard>
 
       {!dashboard.archived && (
-        <ContentSection>
-          <Stack spacing="md">
-            <Switch
-              disabled={!canWrite}
-              label={t`Auto-apply filters`}
-              labelPosition="left"
-              variant="stretch"
-              size="sm"
-              id={autoApplyFilterToggleId}
-              checked={dashboard.auto_apply_filters}
-              onChange={e => handleToggleAutoApplyFilters(e.target.checked)}
-            />
-            {showCaching && isCacheable && (
-              <PLUGIN_CACHING.SidebarCacheSection
-                model="dashboard"
-                item={dashboard}
-                setPage={setPage}
-              />
-            )}
-          </Stack>
-        </ContentSection>
+        <SidesheetCard>
+          <Switch
+            disabled={!canWrite}
+            label={t`Auto-apply filters`}
```

Comment:
Just thinking out loud about the copy, now that I'm looking at it again with fresh eyes. Why not "Automatically apply filters"?

Reply:
These copy suggestions are not blockers